### DISCUSSION TITLE
Flutter: Shimmer effect

### DISCUSSION THREAD

**Discussion by bankatha** [community/community#21310] [Mobile] [answered]
How can create shimmer effect on data loading

**Comment by surajadkhari** [comment, marked as answer]
Official Doc
https://docs.flutter.dev/cookbook/effects/shimmer-loading

Available Package
https://pub.dev/packages/shimmer



SizedBox(
  width: 200.0,
  height: 100.0,
  child: Shimmer.fromColors(
    baseColor: Colors.red,
    highlightColor: Colors.yellow,
    child: Text(
      'Shimmer',
      textAlign: TextAlign.center,
      style: TextStyle(
        fontSize: 40.0,
        fontWeight:
        FontWeight.bold,
      ),
    ),
  ),
);

**Reply by bankatha** [reply]
Thank you would please provide some examples as well
